doc/03-omnifest/02-external.md

```diff
@@ -67,6 +67,66 @@ Into this YAML structure:
 dummy:
 ```
 
+## Example
+
+The following example demonstrates how an external can be used to implement string
+concatenation.
+
+Given the following script called `concat` in `/usr/local/libexec/otk/concat`:
+
+```bash
+#!/usr/bin/env bash
+
+output="$(jq -jr '.tree."otk.external.concat".parts[]' <<< "${1}")"
+echo "{\"tree\":{\"output\":\"$output\"}}"
+```
+
+and the following directive:
+
+```yaml
+and the following directive:
```

Comment:
Duplicate line snuck into the yaml code block.

Reply:
Removed.